src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,10 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        int total = (bet1 > bet2) && (result1 > result2) ? 1 : 0;
+        total = (bet1 < bet2) && (result1 < result2) ? 1 : total;
+        total = (bet1 == bet2) && (result1 == result2) ? 1 : total;
+        total = (result1 + " " + result2).equals(bet1 + " " + bet2) ? 2 : total;
+        return total;
```

Comment:
it's too complicated.Try to solve it without creating redundant variables and with less number of '?' conditions.

Reply:
done